ts/components/session/settings/SessionSettings.tsx

```diff
@@ -574,6 +576,11 @@ export class SettingsView extends React.Component<SettingsViewProps, State> {
 
   private getLinkedDeviceSettings(): Array<LocalSettingType> {
     const { linkedPubKeys } = this.state;
+    const { isSecondaryDevice } = this.props;
+    // tslint:disable-next-line: no-backbone-get-set-outside-model
```

Comment:
Does this rule even apply to the next line?

Reply:
yes, every time we use window.storage.get in typescript tslint complains about that one.
a fix would probably to pass that value as props too to all impacted components

Reply:
I don't see a window.storage.get anywhere in this PR here...

Reply:
You mean `window.textsecure.storage.get`?

Reply:
oops, yes I was not looking at the correct file. the one there https://github.com/loki-project/session-desktop/pull/1011/files/71cf53af15cef07d27511cc71121fb9173b80d1e#diff-fb2ab96e8481edb83c4a0400005af877R18 is needed but not this one